tests/test_operation.py

```diff
@@ -38,12 +38,20 @@ def test_heisenberg(self):
         def h_test(cls):
             "Test a gaussian CV operation."
             log.debug('\tTesting: cls.{}'.format(cls.__name__))
+
+            ww = list(range(cls.num_wires))
+
             # fixed parameter values
             if cls.par_domain == 'A':
-                par = [nr.randn(1,1)] * cls.num_params
+                if cls.__name__ == "Interferometer":
+                    ww = list(range(2))
+                    par = [np.array([[0.83645892-0.40533293j, -0.20215326+0.30850569j],
+                                     [-0.23889780-0.28101519j, -0.88031770-0.29832709j]])]
```

Comment:
Maybe it would be slightly better to generate this with `rand` (and a fixed seed to make the test deterministic), as is done two lines below.

Reply:
I would argue that we should aim for explicit test parameters everywhere, and deprecate use of `np.random`. Even though use of a fixed seed comes with a [cross-platform compatibility guarantee](https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.RandomState.html#numpy.random.RandomState), I find it significantly speeds up test debugging when you can visually compare the input with the expected/resultant output.

Reply:
Ok, then I would suggest to also change the two lines below to explicite numbers.